blazor/scheduler/data-binding.md

```diff
@@ -559,6 +559,50 @@ You can bind Web API data to the scheduler using WebApiAdaptor. The following sa
 }
 ```
 
+### Url adaptor
+
+You can use the [UrlAdaptor](https://blazor.syncfusion.com/documentation/data/adaptors#url-adaptor) of **SfDataManager** when binding data source for remote data. During the initial load of Scheduler, data are fetched from remote data and bound to the Scheduler using the Url property of **SfDataManager**.
+
+The following sample code demonstrates binding data to the Scheduler component through the SfDataManager using UrlAdaptor.
+
+```cshtml
+@using Syncfusion.Blazor
+@using Syncfusion.Blazor.Data;
+@using Syncfusion.Blazor.Schedule
+
+<SfSchedule TValue="AppointmentData" Height="550px" @bind-SelectedDate="@currentDate" Readonly="true">
+    <ScheduleEventSettings TValue="AppointmentData">
+        <SfDataManager Url="http://localhost:54738/Home/LoadData" CrudUrl="http://localhost:54738/Home/UpdateData" CrossDomain="true" Adaptor="Adaptors.UrlAdaptor">
```

Comment:
include the code snippet for updateData and loadData.. means server end code for this operations.

Reply:
Will address this review corrections and add an working sample in separate PR.